nmostesting/mocks/Node.py

```diff
@@ -505,7 +505,7 @@ def staged(version, resource, resource_id):
     """
     # Track requests
     NODE.staged_requests.append({'method': request.method, 'resource': resource, 'resource_id': resource_id,
-                                 'data': request.json})
+                                 'data': request.get_json() if request.is_json else None})
```

Comment:
According to other examples of `get_json` in this repo, and the docs at https://flask.palletsprojects.com/en/2.2.x/api/#flask.Request.get_json, this may work:

```suggestion
                                 'data': request.get_json(silent=True)})
```

Reply:
Yes, this works